deployment/hasura/metadata/databases/AerieMerlin/tables/metadata/plan_snapshot_tags.yaml

```diff
@@ -0,0 +1,48 @@
+table:
+  name: plan_snapshot_tags
+  schema: metadata
+configuration:
+  custom_name: "plan_snapshot_tags"
+object_relationships:
+  - name: plan_snapshot
+    using:
+      foreign_key_constraint_on: snapshot_id
+  - name: tag
+    using:
+      foreign_key_constraint_on: tag_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [snapshot_id, tag_id]
+      check: {}
+  - role: user
+    permission:
+      columns: [snapshot_id, tag_id]
+      check: {"plan_snapshot":{"plan":{"_or":[
+        {"owner":{"_eq":"X-Hasura-User-Id"}},
+        {"collaborators":{"collaborator":{"_eq":"X-Hasura-User-Id"}}}]}}}
```

Comment:
Update permissions aren't applicable, since we have to delete and recreate rows on tag change correct?

Reply:
Can you clarify what you mean by "on tag change"?

Update Permissions aren't applicable b/c this is an association table. You can't "update" an association, it either exists or it doesn't. (And if by some miracle the ids are ever updated, there's a `cascade` on the underlying table)

Reply:
Makes sense. I just wanted to double check that updating these rows wasn't a valid use case, i.e. atomically switching from one tag id to another _could_ be an update, but instead we have the client delete the existing association, and then create a new association.